stacker_blueprints/sns.py

```diff
@@ -23,19 +23,21 @@
 
 
 def queue_policy(sns_arn, sqs_arns):
-    return Policy(
-        Statement=[
+    stmts = []
+    for arn in sqs_arns:
+        stmts.append(
             Statement(
                 Effect="Allow",
                 Principal=Principal("*"),
                 Action=[awacs.sqs.SendMessage],
-                Resource=sqs_arns,
+                Resource=arn,
```

Comment:
It doesn't.  I don't even understand why they accept an array at all, if they aren't going to let you use it.  Really strange design decision on AWS's part.